docs/source/release/v6.11.0/framework.rst

```diff
@@ -10,66 +10,103 @@ Algorithms
+- Binary operations :ref:`Plus <algm-Plus>`, :ref:`Minus <algm-Minus>`, :ref:`Divide <algm-Divide>` and :ref:`Multiply <algm-Multiply>` now support :ref:`Ragged Workspaces <Ragged_Workspace>`.
+- Algorithm :ref:`CompressEvents <algm-CompressEvents>` now has new property ``SortFirst`` that controls whether sorting first happens for compress events.
+  If ``SortFirst=False`` then a different method, that will not sort events first, will be used to compress events which is faster when you have a large number of events per compress tolerance.
+- Algorithm :ref:`ISISIndirectEnergyTransfer <algm-ISISIndirectEnergyTransfer>` has new property ``OutputSuffix`` that will append a suffix to the end of output workspace names.
+- Algorithms :ref:`AlignAndFocusPowderFromFiles <algm-AlignAndFocusPowderFromFiles>` and ref: :ref:`SNSPowderReduction <algm-SNSPowderReduction>` have new property ``MinSizeCompressOnLoad`` for specifying load compression.
+- Algorithm `<algm-NMoldyn4Interpolation>` now uses `scipy.interpolate.RectBivariateSpline` instead of `scipy.interpolate.interp2d`, since `interp2d` has been removed in version 1.14 of `scipy`.
+  See reference documentation here (https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp2d.html).
+- Algorithm :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection-v1>` now supports a radial collimator that restricts scatter points within a small region within the larger sample volume.
+  The algorithm was modified to assign zero weight to tracks where the final scatter is not in a position that allows the final track segment to pass through the collimator toward detectors.
+- Algorithm :ref:`LoadEventAsWorkspace2D <algm-LoadEventAsWorkspace2D>` has new version 2 that adds property ``FilterByTime``.
+- New algorithm :ref:`ScaleInstrumentComponent <algm-ScaleInstrumentComponent>` to scale all detectors in an instrument component around the component's geometrical position.
+
+  .. figure::  ../../images/6_11_release/ScaleInstrumentComponent.png
+     :width: 400px
+
+- Algorithm :ref:`GenerateGroupingSNSInelastic <algm-GenerateGroupingSNSInelastic>` has new input option :ref:`InstrumentDefinitionFile`.
+  Selecting the new ``InstrumentDefinitionFile`` option in instrument drop down menu will create a new field allowing users selecting older instrument definition files.
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces>` has new property ``CheckUncertainty`` to turn off comparing the y-value uncertainties.
+- Algorithm :ref:`ScaleX <algm-ScaleX>` is now 95% faster when using the ``InstrumentParameter`` property.
 
 Bugfixes
 ############
-.. amalgamate:: Framework/Algorithms/Bugfixes
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces-v1>` is now fixed for relative differences of small values.
+- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
```

Comment:
```suggestion
- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has the minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
```